scripts/image_functions.sh

```diff
@@ -6,6 +6,8 @@ function cleanup() {
     find . -maxdepth 5 -name "${IMAGE_NAME}*.d" -type d -exec rm -rf {} +
     echo "Deleting ${IMAGE_NAME}*.qcow2 files"
     find . -maxdepth 5 -name "${IMAGE_NAME}*.qcow2" -type f -exec rm -f {} +
+    echo "Deleting ${IMAGE_NAME}*.raw files"
+    find . -maxdepth 5 -name "${IMAGE_NAME}*.raw" -type f -exec rm -f {} +
```

Comment:
*Not entirely related to the proposed changes but also old code:*
It haunts me a bit that the deletion rule features the wildcard while the retrieval is done with e.g.

`curl -L -o ${IMAGE_NAME}.qcow2`

I tried to give this a good hard stare but didn't find any reason to use the wildcard, which is always a bit risky in a find rm.

You are doing the right thing here using the existing pattern. I'm still pondering should I ask for a change here since it's a bit out of scope of the task.

Reply:
```
imgbuild@imgbuilder-migrated:~$ find . -name "*.raw"
./temp/image.OZ7pMKDb/built/usr/local/cuda-11.2/samples/2_Graphics/marchingCubes/data/Bucky.raw
./temp/image.OZ7pMKDb/built/usr/local/cuda-11.2/samples/2_Graphics/simpleTexture3D/data/Bucky.raw
./temp/image.OZ7pMKDb/built/usr/local/cuda-11.2/samples/2_Graphics/volumeFiltering/data/Bucky.raw
./temp/image.OZ7pMKDb/built/usr/local/cuda-11.2/samples/2_Graphics/volumeRender/data/Bucky.raw
```

I suppose mainly thinking about can it hit stuff like this if IMAGE_NAME is empty due to some weird error. The maxdepth should protect against that as well though.

Reply:
This is the output of the cleanup find commands after a build:
```
imgbuild@imgbuilder-migrated:~$ find . -maxdepth 5 -name "CentOS-7*.d" -type d
./diskimage-builder-scripts.cpouta-devel/CentOS-7.d
imgbuild@imgbuilder-migrated:~$ find . -maxdepth 5 -name "CentOS-7*.qcow2" -type f
imgbuild@imgbuilder-migrated:~$ find . -maxdepth 5 -name "CentOS-7*.raw" -type f
./diskimage-builder-scripts.cpouta-devel/CentOS-7.raw
```
I would say that it is pretty safe to remove the star sign in the find deletion commands.

As a reference, this is the commit that introduced the star sign: https://github.com/CSCfi/diskimage-builder-csc-automation/commit/8c5c02549297ac26f71cc4c1e781a82f422fe2b4 - I think the context has a bit changed, as it seems the commit was trying to cleanup also residuals from previous builds, hence the star

Reply:
Okie, I'm in favor of removing the wildcard and not introducing new ones if it makes sense to you :+1: